### PR TITLE
Sy 1709 fix data fetching on log

### PR DESCRIPTION
**Pull request by emilbon99**
# Issue Pull Request

## Key Information

- **Linear Issue**: [SY-1709](https://linear.app/synnax/issue/SY-1709/fix-data-fetching-on-log)

## Description

Fix data fetching on log.

## Basic Readiness

- [x] I have performed a self-review of my code.
- [x] I have added relevant tests to cover the changes to CI.
- [x] I have added needed QA steps to the [release candidate](/synnaxlabs/synnax/blob/main/.github/PULL_REQUEST_TEMPLATE/rc.md) template that cover these changes.
- [x] I have updated in-code documentation to reflect the changes.
- [x] I have updated user-facing documentation to reflect the changes.

### Data Structures

I have ensured that previous versions of stored data structures are properly migrated to new formats in the following projects:

- [ ] Server
- [ ] Console

### API Changes

The following projects have backwards-compatible APIs:

- [ ] Python Client
- [ ] Server
- [ ] TypeScript Client

### CODE REVIEW

**Review by pjdotson — APPROVED** [review]

**Comment by Lham42** [comment]
Can you add a better pr description about what things you had to change or add? Or specify what the issue needed to be fixed specifically with data fetching on log?

File: cesium/internal/index/domain.go [hunk 1]
```diff
@@ -125,10 +124,9 @@ func (i *Domain) Distance(
 				startToFirstEnd.Lower+gap,
 				startToFirstEnd.Upper+gap,
 			)
-			domainBounds.Upper = iter.Position()
+			alignment = telem.NewAlignmentPair(iter.Position(), uint32(iter.Len()/8))
```
Comment (Lham42) [inline cesium/internal/index/domain.go:127]: Why is the sample index here an eight of the domain length? Could you leave a comment here to clarify

File: cesium/internal/unary/iterator.go [hunk 2]
```diff
@@ -313,16 +354,16 @@ func (i *Iterator) insert(series telem.Series) {
...
-	idxDomain uint32,
+	alignment telem.AlignmentPair,
 	offset telem.Offset,
 	size telem.Size,
 ) (series telem.Series, err error) {
 	series.DataType = i.Channel.DataType
 	series.TimeRange = i.internal.TimeRange().BoundBy(i.view)
 	series.Data = make([]byte, size)
-	inDomainAlignment := uint32(i.Channel.DataType.Density().SampleCount(offset))
+	//inDomainAlignment := uint32(i.Channel.DataType.Density().SampleCount(offset))
```
Comment (Lham42) [inline cesium/internal/unary/iterator.go:364]: do we still need this commented code?

**Review by Lham42 — APPROVED** [review]
approved once you address comments

**Pull request merged**